### PR TITLE
Add InputTable to QST

### PR DESCRIPTION
**Pull request by devinrsmith**

### CODE REVIEW

File: java-client/flight-examples/src/main/java/io/deephaven/client/examples/AddToInputTable.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    protected void execute(FlightSession flight) throws Exception {
+        final ColumnHeader<Instant> header = ColumnHeader.ofInstant("Timestamp");
+
+        final TableSpec timestamp = InMemoryAppendOnlyInputTable.of(TableHeader.of(header));
+        final TableSpec timestampLastBy =
+                timestamp.by(Collections.emptyList(), Collections.singletonList(Aggregation.AggLast("Timestamp")));
+
+        final List<TableHandle> handles = flight.session().batch().execute(Arrays.asList(timestamp, timestampLastBy));
+        try (
+                final TableHandle timestampHandle = handles.get(0);
```
[the remark this answers is not in the document]
Comment (devinrsmith) [reply java-client/flight-examples/src/main/java/io/deephaven/client/examples/AddToInputTable.java:35]: Yeah - I somewhat like having try resources aligned.

File: java-client/flight-examples/src/main/java/io/deephaven/client/examples/AddToInputTable.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                final TableHandle timestampLastByHandle = handles.get(2)) {
```
[the remark this answers is not in the document]
Comment (devinrsmith) [reply java-client/flight-examples/src/main/java/io/deephaven/client/examples/AddToInputTable.java:36]: Oops - that  was from previous iter :) - will fix.

File: java-client/flight/src/main/resources/META-INF/services/io.deephaven.client.impl.SchemaBytes [hunk 3]
```diff
@@ -0,0 +1 @@
+io.deephaven.client.impl.SchemaBytesImpl
```
[the remark this answers is not in the document]
Comment (devinrsmith) [reply java-client/flight/src/main/resources/META-INF/services/io.deephaven.client.impl.SchemaBytes:1]: sure.

File: java-client/flight/src/main/java/io/deephaven/client/impl/FieldAdapter.java [hunk 4]
```diff
@@ -1,108 +1,123 @@
...
+        return field(name, MinorType.FLOAT8.getType(), "double");
+    }
+
+    public static Field stringField(String name) {
+        return field(name, MinorType.VARCHAR.getType(), "java.lang.String");
     }
 
-    private Field field(FieldType type) {
+    public static Field instantField(String name) {
+        return field(name, MinorType.TIMESTAMPNANO.getType(), "io.deephaven.db.tables.utils.DBDateTime");
```
[the remark this answers is not in the document]
Comment (devinrsmith) [reply java-client/flight/src/main/java/io/deephaven/client/impl/FieldAdapter.java:81]: So - neither Instant nor DBDateTime has a "timezone" afaict?

Under the covers, it looks like MinorType.TIMESTAMPNANO is defined as:

```java
    // time in nanosecond from the Unix epoch, 00:00:00.000000000 on 1 January 1970, UTC.
    TIMESTAMPNANO(new Timestamp(org.apache.arrow.vector.types.TimeUnit.NANOSECOND, null)) {
```

I'm confused about that comment on NANO_SINCE_EPOCH_TYPE given the above...

**Review by nbauernfeind — APPROVED** [review]

**Pull request merged**